configure.ac

```diff
@@ -293,7 +293,7 @@ case "$with_platform" in
 	AC_SUBST(XTENSA_CORE)
 
 	AC_DEFINE([CONFIG_APOLLOLAKE], [1], [Configure for Apollolake])
-	AC_DEFINE([CONFIG_BOOT_LOADER], [1], [Configure Boot Loader])
+	AC_DEFINE([CONFIG_KABYLAKE], [1], [Configure for Kabylake])
```

Comment:
Why both CONFIG_KABYLAKE and CONFIG_APOLLOLAKE, in kconfig I made this as choice type so it won't be even possible to have both. If it's just some kind of kbl-only feature please name it CONFIG_KABYLAKE_BOOTLOADER or something like that, because by looking at name I guess it's mostly like apl platform but just kbl bootloader is different.

Reply:
It's there due to some feature overlap and also a little differentiation, but I was waiting to apply this after kconfig merge. It may actaully be better for us to define CAVS_1_5, CAVS_1_8 etc and let the Kconfig target i.e. Apollolake select CAVS_version etc.

Reply:
@lgirdwood 
Yes, this is exactly what we talked about with Janusz recently, to have a solid definition of cavs_hw_ver including set of supported features for each cavs hw version first. And then build the platform definitions based on the selected and parametrized cavs_hw + some other platform/board specific things if any.

Reply:
ok, sounds like we are all in agreement and can do this after kconfig merge